### ISSUE TITLE
mixed use of training and validation accuracies

### ISSUE THREAD

**Issue by dyigitpolat** [dyigitpolat/mimarsinan#40]
mixed use of training and validation accuracies causes the adaptation process to work in unpredictable ways especially when training and validation accuracies are different because of "overfitting" or certain "regularization" methods. 

this issue is linked to https://github.com/dyigitpolat/mimarsinan/issues/37

**Comment by dyigitpolat** [comment]
start fixing by never using validation accuracy for target accuracy propagation

**Comment by dyigitpolat** [comment]
fixed instead by only using validation accuracy.
